metadata-ingestion/src/datahub/ingestion/source/sql/two_tier_sql_source.py

```diff
@@ -71,6 +71,10 @@ def __init__(self, config, ctx, platform):
         super().__init__(config, ctx, platform)
         self.config: TwoTierSQLAlchemyConfig = config
 
+    def get_db_schema(self, dataset_identifier: str) -> Tuple[Optional[str], str]:
```

Comment:
When we do the refactoring, let's put identifiers in a real data class instead of joining into a string in one place and then splitting in other places